### DISCUSSION TITLE
Not received Student Developer Pack

### DISCUSSION THREAD

**Discussion by aboutzack** [community/community#147155] [GitHub Education]
### Hi everyone,

I got approved for my GitHub Student Developer Pack on December 12th. However, it’s already December 16th, and I have not yet received access to the pack. I am really excited to start using the benefits and would appreciate any help with this issue.

Thank you! :)

**Comment by davevad93** [comment]
Hi @aboutzack ,

What's the status of your application?

If the label is green and states "`Congrats! Your request for @aboutzack was approved and your benefits will be available soon.` that means you have to wait. It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

If the label is purple and states "`Congrats! Your request for @aboutzack was approved and your benefits are now available.`", that means you should have access to your benefits. Try first going to the [pack offers page](https://education.github.com/pack), scroll down to the "***All offers***" section and select one of the available offers, for example [Copilot](https://education.github.com/pack/redeem/copilot-student), if you can redeem it you are fine, if you can't, I suggest you to raise a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.

For further reference, take a look at the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352):

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

And also the following discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #144741, #145995

**Reply by aboutzack** [reply]
The label is green. Looks like I'll have to wait some more.

**Reply by 27bchaos** [reply]
That sucks.

**Comment by samus-aran** [comment]
Hey @aboutzack 👋 

We can confirm you have been successful and your academic benefits have been granted. 🎉  Just to provide additional context that your application was submitted, approved and granted within 72 hours exactly 😅 We usually ask members to raise queries on their benefits after 5 days as stated on our FAQs (which are linkable from your application page).

https://github.com/orgs/community/discussions/111352#user-content-how-long-after-ive-been-approved-will-i-receive-my-academic-benefits
